src/Client.js

```diff
@@ -155,7 +155,10 @@ var values = require('./values')
  * @param {?fetch} options.fetch
  *   a fetch compatible [API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) for making a request
  * @param {?number} options.queryTimeout
- *   Sets the maximum amount of time (in milliseconds) for query execution on the server,
+ *   Sets the maximum amount of time (in milliseconds) for query execution on the server
+ * @param {?number} options.http2SessionIdleTime
+ *   Sets the maximum amount of time (in milliseconds) an HTTP2 session may live
+ *   when there's no activity. Only applicable for NodeJS environment (when http2 module is used), 500 by default.
  */
 function Client(options) {
   options = util.applyDefaults(options, {
```

Comment:
@faunaee it looks good, I like the idea, especially if it could fix your docs tests and avoid the delay. One concern is that we need to verify if it's NodeJS env to access the "process" property but no worries, I'll fix and test it tomorrow. Thank you, Ewan!

Reply:
@faunaee updated to pull from env variable by default. I use FAUNADB_HTTP2_SESSION_IDLE_TIME name for the env variable to match `Client`'s option name